src/leo-client-app/src/components/EditSchedules.tsx

```diff
@@ -0,0 +1,243 @@
+// import React from 'react';
+// // import './EditSchedulePage.css'; // Your stylesheet filename
+
+// interface Command {
+//   id: number;
+//   name: string;
+//   operator: string;
+//   created: string;
+// }
+
+// // Example commands data
+// const commands: Command[] = [
+//   { id: 1, name: 'setUp', operator: 'operator1', created: 'Jan 1 2023' },
+//   { id: 2, name: 'runDiagnostics', operator: 'operator1', created: 'Jan 1 2023' },
+//   { id: 3, name: 'checkSensor', operator: 'operator1', created: 'Jan 1 2023' },
+//   { id: 4, name: 'loadData', operator: 'operator2', created: 'Jan 2 2023' },
+//   { id: 5, name: 'tearDown', operator: 'operator1', created: 'Jan 1 2023' },
+//   // ... more commands
+// ];
+
+// const EditSchedulePage: React.FC = () => {
+//   return (
+//     <div className="edit-schedule-page">
+//       <div className="header">
+//         <button className="back-button">{"<"}</button>
+//         <h2>Edit Schedule</h2>
+//         <button className="close-button">{"X"}</button>
+//       </div>
+//       <h3>Jan 09 Schedule</h3>
+//       <table>
+//         <thead>
+//           <tr>
+//             <th>id</th>
+//             <th>Command</th>
+//             <th>Operator</th>
+//             <th>Created</th>
+//           </tr>
+//         </thead>
+//         <tbody>
+//           {commands.map(command => (
+//             <tr key={command.id}>
+//               <td>{command.id}</td>
+//               <td>{command.name}</td>
+//               <td>{command.operator}</td>
+//               <td>{command.created}</td>
+//             </tr>
+//           ))}
+//         </tbody>
+//       </table>
+//       <div className="actions">
+//         <button className="add-command">add new command</button>
+//         <button className="see-logs">see logs</button>
+//       </div>
+//       <div className="footer">
+//         <button className="save-button">Save</button>
+//         <button className="cancel-button">Cancel</button>
+//       </div>
+//     </div>
+//   );
+// };
+
+// export default EditSchedulePage;
+
+import '../components/Scheduler.css'; // Import the CSS styles
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import EditSchedulePage from "@/components/EditSchedules";
+import { useRouter } from "next/router";
+
+
+import { sendCommandSchedule } from "@/constants/api";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import React, { useEffect, useState } from "react";
```

Comment:
nit: import { React, useEffect, useState }

Reply:
writing the import statement in the way you suggested resulted in an import error. i chose to switch back to this way for it to pass since it's not a big difference.